From 0ecfab89d1754b0b7e32b3964b40663d1810cd62 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Mon, 22 Mar 2010 14:04:09 +1000
Subject: [PATCH 24/37] xfree86: remove if 1 from the dawn of time.

7+ years for an if 1 should be enough to just admit that there's no other
option.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
Reviewed-by: Fernando Carrijo <fcarrijo@yahoo.com.br>
Reviewed-by: Corbin Simpson <MostAwesomeDude@gmail.com>
---
 hw/xfree86/os-support/shared/posix_tty.c |   10 ----------
 1 files changed, 0 insertions(+), 10 deletions(-)

diff --git xorg-server-1.7.99.902/hw/xfree86/os-support/shared/posix_tty.c xorg-server-1.7.99.902/hw/xfree86/os-support/shared/posix_tty.c
index bf1bb03..4ba8599 100644
--- xorg-server-1.7.99.902/hw/xfree86/os-support/shared/posix_tty.c
+++ xorg-server-1.7.99.902/hw/xfree86/os-support/shared/posix_tty.c
@@ -137,19 +137,9 @@ xf86OpenSerial (pointer options)
 
 	if (!isatty (fd))
 	{
-#if 1
 		/* Allow non-tty devices to be opened. */
 		xfree(dev);
 		return (fd);
-#else
-		xf86Msg (X_WARNING,
-			 "xf86OpenSerial: Specified device %s is not a tty\n",
-			 dev);
-		SYSCALL (close (fd));
-		errno = EINVAL;
-		xfree(dev);
-		return (-1);
-#endif
 	}
 
 	/* set up default port parameters */
-- 
1.7.0.2

